### ISSUE TITLE
Rendering of help text of "models" fails in development version

### ISSUE THREAD

**Issue by nomennescio** [factor/factor#2361]
The help text for class "model" is corrupted in the development version. When compared to the correct help text of the release version, formatting has changed from:
```
HELP: model
{ $class-description "A mutable cell holding a single value. When the value is changed, a sequence of connected objects are notified. Models have the following slots:"
    { $list
        { { $slot "value" } " - the value of the model. Use " { $link set-model } " to change the value." }
```
to:
```
HELP: model
{ $class-description "A mutable cell holding a single value. When the value is changed, a sequence of connected objects are notified. Models have the following slots:"
    { $slots
        { "value" "the value of the model. Use " { $link set-model } " to change the value." }
```
The incorrect version does not show any links and overwrites text in the table.

**Comment by mrjbq7** [comment]
This looks like it was fixed in 258d7e05d66, which was 5 months ago.

**Comment by nomennescio** [comment]
Duh! I was sure I was looking at the latest development version, apparently not...
